src/cmdlib.sh

```diff
@@ -102,6 +102,14 @@ preflight() {
         fatal "Your umask is unset, please use umask 0022 or so"
     fi
 
+    if ! has_privileges && [ -n "${ISEL}" ]; then
```

Comment:
I think we can just delete this code, it came in bba228c4ca172a05270b2e938953d9d27691312d and is completely irrelevant now, we clearly aren't going to support a RHEL7 cosa (or even a RHEL8).